### PR TITLE
Add New Plugin (Notion Plugin)

### PR DESCRIPTION
**Pull request by AminSallah**
A new Flow Launcher Plugin to interface with [Notion](https://www.notion.so/)

✨ Key Features:
Search: Effortlessly, High performance search and access Notion pages directly from Flow Launcher. 
Create: Instantly create new Notion entries. 
Edit: Effortlessly edit Notion pages with a few keystrokes. 
Delete: Deleting Notion entries on the fly. and much more..

Repository Link: [Flow Launcher Notion Plugin](https://github.com/AminSallah/Flow.Launcher.Plugin.Notion)
 Let me know if you need anything else

### CODE REVIEW

**Comment by jjw24** [comment]
Hey @AminSallah thanks for creating this great plugin. Could I please ask you to use GitHub action in your repo to publish your release please. Feel free to have a look at how the Window Walker plugin is doing it as an example:
https://github.com/taooceros/Flow.Plugin.WindowWalker/blob/master/.github/workflows/publish-action.yml

**Review by jjw24 — CHANGES_REQUESTED** [review]
Please see above comment

**Comment by AminSallah** [comment]
Hi @jjw24 thanks for your suggestion, I've implemented the GitHub action to publish the release.
if there's anything else you'd like to share, feel free to let me know.

File: plugins.json [hunk 1]
```diff
@@ -2134,5 +2134,19 @@
...
+        "Description": "A Plugin For Search, Create, Edit and Delete Notion Pages.",
+        "Author": "Amin Salah",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/AminSallah/Flow.Launcher.Plugin.Notion",
+        "UrlDownload": "https://github.com/AminSallah/Flow.Launcher.Plugin.Notion/releases/download/Notion/Flow.Launcher.Plugin.Notion.zip",
+        "UrlSourceCode": "https://github.com/AminSallah/Flow.Launcher.Plugin.Notion/tree/main/src",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/AminSallah/Flow.Launcher.Plugin.Notion@main/src/Images/app.png",
+        "DateAdded": "2024-02-04T05:17:00Z",
+        "LatestReleaseDate": "2024-02-04T04:43:28Z"
```
Comment (Garulf) [inline plugins.json:2150]:
```suggestion
```

These lines will be added by the CI no need to add them manually.

**Review by Garulf — CHANGES_REQUESTED** [review]

**Review by Garulf — APPROVED** [review]
Everything looks good now! I am not a notion user but the plugin looks amazing!

File: plugins.json [hunk 2]
```diff
@@ -2134,5 +2134,17 @@
...
+        "ID": "BDC409081E7F4D879FC1FE8BC7F9D606",
+        "Name": "Notion",
+        "Description": "A Plugin For Search, Create, Edit and Delete Notion Pages.",
+        "Author": "Amin Salah",
+        "Version": "1.0.0",
+        "Language": "csharp",
+        "Website": "https://github.com/AminSallah/Flow.Launcher.Plugin.Notion",
+        "UrlDownload": "https://github.com/AminSallah/Flow.Launcher.Plugin.Notion/releases/download/Notion/Flow.Launcher.Plugin.Notion.zip",
+        "UrlSourceCode": "https://github.com/AminSallah/Flow.Launcher.Plugin.Notion/tree/main/src",
+        "IcoPath": "https://cdn.jsdelivr.net/gh/AminSallah/Flow.Launcher.Plugin.Notion@main/src/Images/app.png",
```
Comment (Garulf) [inline plugins.json:2148]:
```suggestion
        "IcoPath": "https://cdn.jsdelivr.net/gh/AminSallah/Flow.Launcher.Plugin.Notion@main/src/Images/app.png"
```

**Review by Garulf — APPROVED** [review]

**Review by Garulf — APPROVED** [review]

**Comment by AminSallah** [comment]
Thank you @Garulf, for your words! I was pondering the best way to update the url downloading for other versions or refresh it to use release made by github action. Should I initiate a new pull request, or is there another approach?

**Comment by Garulf** [comment]
Our CI should pick up any updated releases automatically.

**Comment by AminSallah** [comment]
Thanks @Garulf for your guidance. I appreciate your time.

**Pull request merged**